app/src/main/java/org/dhis2/uicomponents/map/model/RelationshipUiComponentModel.kt

```diff
@@ -11,7 +11,7 @@ data class RelationshipUiComponentModel(
     val from: TeiMap,
     val to: TeiMap,
     val canBeDeleted: Boolean? = true
-)
+) : CarouselItemModel
```

Comment:
why extending from carouselItem? I think they are different things. One model represent the carousel, the other model represent lines in map